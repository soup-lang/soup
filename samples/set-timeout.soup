f: (x):
    setTimeout(():
        console.log(1)
        console.log(0)
        console.log(x)
    , 10)
    console.log('start')

f(42)
