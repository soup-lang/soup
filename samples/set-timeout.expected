start
1
0
42
